src/steps/mdx-to-gridtables.ts

```diff
@@ -65,6 +65,24 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
   // for loop since we mutate in the loop
   for (let i = 0; i < mdast.children.length; i += 1) {
     const node = mdast.children[i];
+
+    // handle raw <hr /> conversion
+    if (node.type === 'mdxJsxFlowElement' && node.name === 'hr') {
```

Comment:
A small suggestion - we can remove redundancy by moving lines 68 - 85 under line 90 and change condition to: 

```suggestion
    if (node.name === 'hr') {
```